packages/app/src/settings/project/ConfigCode.cy.tsx

```diff
@@ -75,8 +75,10 @@ describe('<ConfigCode />', () => {
         .should('contain.text', 'plugin')
       })
 
-      // Take a snapshot of the last case
-      cy.percySnapshot()
+      cy.get('.v-popper__popper--shown').should('have.length', 1).then(() => {
+        // Take a snapshot after the other tooltips have closed
+        cy.percySnapshot()
```

Comment:
Nit: Could add this inside the `each` after the `realHover`

Reply:
I don't think we need to take a snapshot for each tooltip, just one. What do you think?

Reply:
Ah snap I meant to comment on the `cy.get('.v-popper__popper--shown').should('have.length', 1)`, if you bumped that up it would enforce each interaction to deal with only one tooltip, then you wouldn't have to add it below.

Reply:
ah right, good call